styles.css

```diff
@@ -0,0 +1,44 @@
+body {
+    background-color: #3b3b4f;
+    color: #ffffff;
+    font-family: tahoma;
+    min-width: 350px;
+    width: 100%;
+    max-width: 500px;
+    height: 100vh;
+    margin: auto;
+}
+
+#survey-form {
+    padding: 20px 20px
+}
+
+h1,
+p {
+    margin: 1em auto;
```

Comment:
why using em?

Reply:
we use 'em' if we want to make our design flexible and scaleable according to user preferences.
we should use 'em' with margin, padding and font-sizes for better and responsive design.
we can use 'px' for fixed sizes i.e: borders and icons etc.